modules/users/client/components/utils/filters.js

```diff
@@ -0,0 +1,7 @@
+export const limitTo = (text, length) => {
+  return text.substring(0, length);
+};
+
+export const plainTextLength = (text) => {
+  return text && typeof(text) === 'string' ? String(text).replace(/&nbsp;/g, ' ').replace(/<[^>]+>/gm, '').trim().length : 0; // eslint-disable-line angular/typecheck-string
```

Comment:
> `eslint-disable-line angular/typecheck-string`

Hmm, it's surprising that Angular linter is trying to lint here. 

No need to change anything, I'm just noting to look into this when working again on eslint stuff: https://github.com/Trustroots/trustroots/pull/932